### PR TITLE
Jordan/clipboard

### PR DESCRIPTION
**Pull request by jordanlleslie**
## Describe your changes
Added copy to pickup/dropoff to clipboard, there is no UI for this but you can tap on the location to copy to clipboard. Includes expo-clipboard native module so use the latest dev build to review

## This PR is for:

- [x] User Story/Feature
- [ ] Bug fix
- [ ] Refactoring
- [ ] Maintenance

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: components/posts/PostCardShared.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export function PostCardGuts({ post, isProfile, isMatched, userInfo }: PostCardG
                     ) : (
                         <Right color={color.p} height={20} />
                     )}
-                    <Text textStyle="header" styleSize="s" style={{ color: color.p }}>
+                    <Text textStyle="header" styleSize="s">
```
Comment (willtheodore) [inline components/posts/PostCardShared.tsx:41]: why change the color here?

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
         "expo": "^48.0.4",
         "expo-asset": "~8.9.1",
         "expo-build-properties": "~0.6.0",
+        "expo-clipboard": "^4.1.2",
```
Comment (willtheodore) [inline package.json:36]: does expo doctor and yarn install --check-files run clean?

**Review by willtheodore — COMMENTED** [review]
Couple of small things to check on but looks good

File: components/posts/PostCardShared.tsx (hunk 1 above)
Comment (jordanlleslie) [reply components/posts/PostCardShared.tsx]: accident from copy pasting lines oops

File: package.json (hunk 2 above)
Comment (jordanlleslie) [reply package.json:36]: yes, both run clean

**Review by willtheodore — APPROVED** [review]

**Pull request merged**